website/data/docs-nav-data.json

```diff
@@ -1612,11 +1612,6 @@
           },
           {
             "title": "Enrollment over Secure Transport (EST)",
-            "badge": {
```

Comment:
huh, should this be replaced with the enterprise badge? I guess we couldn't have both before?

Reply:
Oh, good call.